_docs/privacy/web-tracking-protections.md

```diff
@@ -292,16 +292,16 @@ We review user-reported Broken Site Reports daily, and work to address these usa
 
-The Fire Button erases traces of websites that you’ve visited (like cookies, website caches, and favicons), browsing session information (like open tabs, back button history, and visited URLs), and permissions you’ve granted to websites (like geolocation, camera, and microphone settings). For additional control, you can also make specific sites partially immune to the Fire Button by “Fireproofing” them. Fireproofing a website keeps some helpful data around, like 1st-party cookies and storage that would otherwise be cleared out by the Fire Button, so you can stay signed in.
+The Fire Button erases traces of websites that you’ve visited (like cookies, website caches, and favicons), browsing session information (like open tabs, back button history, and visited URLs), and permissions you’ve granted to websites (like geolocation, camera, and microphone settings). Our apps offer additional control, where you can also make specific sites partially immune to the Fire Button by “Fireproofing” them. Fireproofing a website keeps some helpful data around, like 1st-party cookies and storage that would otherwise be cleared out by the Fire Button, so you can stay signed in.
 
 From DuckDuckGo app Settings, you can manage your Fireproof Sites and choose whether you’d like to automatically clear tabs and data on restart. It’s important to note that the Fire Button will not clear the following: 1st-party cookies and storage for Fireproof Sites, bookmarks, downloaded files, or DuckDuckGo Search settings. This type of protection is not offered in most popular browsers by default.
 
 | Platform          | Support                                                                                                                                                   |
 | ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Firefox extension | Unsupported                                                                                                                                               |
-| Chrome extension  | Unsupported                                                                                                                                               |
-| Edge extension    | Unsupported                                                                                                                                               |
-| Opera extension   | Unsupported                                                                                                                                               |
+| Chrome extension  | Clears all browsing history and data. Can also clear data for a single site or for time periods. History and tab clearing can be disabled in Settings.                                                                                                                                   |
```

Comment:
hm, should we also say that extensions don't support fireproofing for now? Or is it enough that we hint it out in the main description above?

Reply:
We don't specify that we **do** support it in the table entries for the apps, so I think the description explanation is probably enough here. Any objections, @alohaas ?

Reply:
Agree with @laghee. Let's merge now so we don't fall too far behind the rollout. Description is clear about support, but we can revisit additional mentions in the table and do a follow-up if needed.